index.html

```diff
@@ -53,5 +53,142 @@ <h1 id="myName">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+            </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-me">
+          <h2 class="about-title">About me</h2>
+          <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          <button class="resume">Get my resume</button>
+        </div>
+         <div class="about-img"><img src="img/Frame.png"> </div> 
+         <hr class="divider">
+      </section>
+      <section class="profession-tools">
+
+          <div class="languages">
+              <div class="toolsTitle"> 
+                <img id="titles" src="img/Rectangle 55.png" alt="Languages">
+                <h1 >Language</h1>
+              </div>
+              <ul class="tools-details">
+                <li class="detail">JavaScript</li>
+                <li class="detail">Ruby</li>
+                <li class="detail">HTML</li>
+                <li class="detail">CSS</li>
+              </ul>
+          </div>
+
+          <div class="frameworks">
+            <div class="toolsTitle"> 
+              <img id="titles" src="img/Rectangle 56.png" alt="Languages">
+              <h1>Framework</h1>
+            </div>
+            <ul class="tools-details">
+              <li class="detail">React. js</li>
+              <li class="detail">Ruby on rails</li>
+              <li class="detail">RSpec</li>
+              <li class="detail">Capybara</li>
+            </ul>
+        </div>
+
+        <div class="skills">
+          <div class="toolsTitle"> 
+            <img id="titles" src="img/Rectangle 57.png" alt="Languages">
+            <h1>Skills</h1>
+          </div>
+          <ul class="tools-details">
+            <li class="detail">Database Managment</li>
+            <li class="detail">Version Control</li>
+            <li class="detail">CLI</li>
+            <li class="detail">API Design</li>
+            <li class="detail">Web Development</li>
+          </ul>
```

Comment:
- [ ] Although you implement the list but all the lists are not rendering please stick to the design as much as possible. There is more spacing after the About section and the page is not responsive. Please make sure that all list are render according to the design.
- [ ] Also adjust the spacing of the list as it is on center of every heading. Skills is not rendering on the page pleae fix this issue.
### Your design
![his](https://user-images.githubusercontent.com/81727988/236491757-ca1086fd-dc71-4136-9ad0-22641d6b17d9.png)


### Figma design
![figma](https://user-images.githubusercontent.com/81727988/236491811-f6625acf-50db-4aca-a60c-72ee8278a64b.png)